zerver/tests/test_realm_playgrounds.py

```diff
@@ -88,6 +88,27 @@ def test_invalid_params(self) -> None:
         resp = self.api_post(iago, "/api/v1/realm/playgrounds", payload)
         self.assert_json_error(resp, 'Missing the required variable "code" in the URL template')
 
+    def test_add_realm_playground_validation(self) -> None:
+        iago = self.example_user("iago")
+
+        # Test to check that spaces are rejected
+        payload = {
+            "name": "Bad Name",
+            "pygments_language": "rust lang",
+            "url_template": "https://example.com",
+        }
+        resp = self.api_post(iago, "/api/v1/realm/playgrounds", payload)
```

Comment:
I think `result` would be a better variable name than `resp` and also we use `result` usually in other places as well.